docker/sentinel-compose.yml

```
@@ -0,0 +1,105 @@
+      redis-server /data/sentinel.conf --sentinel
+
+  redis-server-1:
+    image: muhammadqadora/falkor-redis
+    hostname: redis-server-1
+    container_name: redis-server-1
+    ports:
+      - 6380:6380
+    networks:
+      - redis-sentinel
+    command:
+    - bash
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)